src/services/twilio.ts

```diff
@@ -0,0 +1,53 @@
+import { getTwilioConfig } from "@config/app"
+import { UnknownPhoneProviderServiceError } from "@domain/phone-provider"
+import { baseLogger } from "@services/logger"
+import twilio from "twilio"
+
+export const TwilioClient = (): IPhoneProviderService => {
+  const client = twilio(getTwilioConfig().apiKey, getTwilioConfig().apiSecret, {
+    accountSid: getTwilioConfig().accountSid,
+  })
+
+  const sendText = async ({ body, to, logger }) => {
+    const twilioPhoneNumber = getTwilioConfig().twilioPhoneNumber
+    try {
+      await client.messages.create({
+        from: twilioPhoneNumber,
+        to,
+        body,
+      })
+    } catch (err) {
+      logger.error({ err }, "impossible to send text")
+      return new UnknownPhoneProviderServiceError(err)
+    }
+
+    logger.info({ to }, "sent text successfully")
+    return true
+  }
+
+  const getCarrier = async (phone: PhoneNumber) => {
```

Comment:
same here (signature)

Reply:
the signature is the same?

Reply:
please check this comment https://github.com/GaloyMoney/galoy/pull/730#discussion_r757807499